docs/sources/flow/reference/components/faro.receiver.md

```diff
@@ -0,0 +1,267 @@
+---
+aliases:
+- /docs/grafana-cloud/agent/flow/reference/components/faro.receiver/
+- /docs/grafana-cloud/monitor-infrastructure/agent/flow/reference/components/faro.receiver/
+- /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/reference/components/faro.receiver/
+canonical: https://grafana.com/docs/agent/latest/flow/reference/components/faro.receiver/
+title: faro.receiver
```

Comment:
```suggestion
title: faro.receiver
description: Learn about the faro.receiver
```

We need to add the description: metadata in all files. It doesn't have to be elaborate, just present with some simple text.